### PR TITLE
Users Options - passwords

### PR DESCRIPTION
**Pull request by brianteeman**
#### Summary of Changes

Cosmetic change to move the options for passwords to their own tab in com_users options and to change the strings - this should improve usability and remove confusion
#### Testing Instructions

Apply the PR and go got com_users options and see the new tab

After the PR it should look like the following

jul4 1

kpll

### CODE REVIEW

File: administrator/components/com_users/config.xml [hunk 1]
```diff
@@ -131,7 +139,6 @@
 			step="1"
 			default="1">
 			</field>
-
```
Comment (wojsmol) [inline administrator/components/com_users/config.xml]: Revert this change, see https://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md

File: administrator/components/com_users/config.xml [hunk 2]
```diff
@@ -142,7 +149,6 @@
 			step="1"
 			default="4">
 			</field>
-
```
Comment (wojsmol) [inline administrator/components/com_users/config.xml]: Revert this change, see https://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md

File: administrator/components/com_users/config.xml [hunk 3]
```diff
@@ -153,7 +159,6 @@
 			step="1"
 			default="0">
 			</field>
-
```
Comment (wojsmol) [inline administrator/components/com_users/config.xml]: Revert this change, see https://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md

**Comment by zero-24** [comment]
I have tested this item :white_check_mark: successfully on 7340be3305ba45d3f99c13ad4abfa05a53b537cf<br /><br />looks good :+1: Just one tab to much :P https://github.com/brianteeman/joomla-cms/pull/2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10941">issues.joomla.org/joomla-cms/10941</a>.</sub>

**Comment by ggppdk** [comment]
I have tested this item :white_check_mark: successfully on 027848067432d3145e2d2c3bea21680aa2d2d363<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10941">issues.joomla.org/joomla-cms/10941</a>.</sub>

**Comment by zero-24** [comment]
I have tested this item :white_check_mark: successfully on 027848067432d3145e2d2c3bea21680aa2d2d363<br /><br />Still, Thanks! ;)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10941">issues.joomla.org/joomla-cms/10941</a>.</sub>

**Comment by zero-24** [comment]
Thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10941">issues.joomla.org/joomla-cms/10941</a>.</sub>
